gapic/src/main/com/google/gapid/perfetto/models/FrameEventsTrack.java

```diff
@@ -160,7 +196,7 @@ private static String sliceSql(long id) {
 
   public ListenableFuture<List<Slice>> getSlices(TimeSpan ts, int minDepth, int maxDepth) {
     return transform(qe.query(sliceRangeSql(ts, minDepth, maxDepth)),
-        res -> res.list(($, row) -> buildSlice(row, ArgSet.EMPTY)));
+        res -> res.list(($, row) -> buildSlice(row, ArgSet.EMPTY, null)));
```

Comment:
Nit: personal opinion but I think an empty map might be better. The same for null below.

Reply:
We check for null in the selection view and its kinda easier this way. There is a solid difference between an instant event and a displayed frame slice.